lib/cmd/fh3/credential/create.js

```diff
@@ -0,0 +1,105 @@
+    'certificateAndroid': i18n._("Param valid for Android platform. A Distribution Certificate is used to build Release Apps for submission to the Android Marketplace. When you submit an App to the Android Marketplace, your certificate is used to identify you as the developer of the App. Path for the certificate file. (E.g --certificate:/Users/user/certificate)"),
+    'certificateIos': i18n._("Param valid for IOS platform. Your iOS developer certificate file issued by Apple. You can download this file from your iOS developer portal. Path for the certificate file. (E.g --certificate:/Users/user/certificate)"),
+    'provisioning': i18n._("Param valid for IOS platform. A provisioning profile for a particular App ID downloaded from the iOS developer portal. The type of the provisioning profile should match the type selected above. A Distribution Certificate is used to build Release Apps for submission to the Android Marketplace. When you submit an App to the Android Marketplace, your certificate is used to identify you as the developer of the App Path for the provisioning file. (E.g --provisioning:/Users/user/provisioning)")
+  },
+  'url' : '/box/api/credentials',
+  'method' : 'post',
+  'customCmd' : function(params,cb) {
+    if (params.platform && params.platform !== 'ios' && params.platform !== 'android') {
+      return cb(i18n._('Invalid parameter for platform:') + params.platform);
+    }
+
+    var dataObject = [{
+      name: "bundleName",
+      value: params.bundleName
+    }, {
+      name: "platform",
+      value: params.platform
+    },{
+      name: "bundleType",
+      value: 'release'
+    }];
+
+    if (params.env) {
+      dataObject.push({
+        name: "environments",
+        value: params.env
+      });
+    }
+
+    if (params.platform === 'ios' || params.platform === 'android') {
+      if (!params.privateKey) {
+        return cb(i18n._('Missed parameter --privateKey.'));
+      }
+
+      dataObject.push({
+        name: "privatekey",
+        value: fs.createReadStream(params.privateKey),
+      });
+    }
+
+    if (params.platform === 'ios') {
+      if (params.bundleType && params.bundleType !== 'release' && params.bundleType !== 'distribution' &&   params.bundleType !== 'debug') {
+        return cb(i18n._('Invalid parameter for --bundleType :') + params.bundleType);
+      }
+
+      if (!params.provisioning) {
+        return cb(i18n._('Missed parameter --provisioning.'));
+      }
+
+      if (!params.certificateIos) {
+        return cb(i18n._('Missed parameter --certificateIos.'));
+      }
+
+      dataObject.push({
+        name: "certificate",
+        value: fs.createReadStream(params.certificateIos),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)